templates/flavors/default/patches/kubeadm-config-template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+kind: KubeadmConfigTemplate
+metadata:
+  name: "${CLUSTER_NAME}-md-0"
```

Comment:
Can we add something similar for the `machinepool` flavor and for any other worker nodes that don't come from the `default` flavor?

Reply:
I will follow up with another PR.